src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -786,6 +823,8 @@ Events.on(playbackManager, 'playerchange', function () {
 bindToPlayer(playbackManager.getCurrentPlayer());
 
 document.addEventListener('viewbeforeshow', function (e) {
+    lyricPageActive = appRouter.currentRouteInfo.path === '/Lyrics';
```

Comment:
Quick fix. Lyrics toggling doesn't work.
```suggestion
    lyricPageActive = appRouter.currentRouteInfo.path.toLowerCase() === '/lyrics';
```